docker/config/devnet/tendermint/config.toml

```diff
@@ -0,0 +1,91 @@
+proxy_app = "tcp://127.0.0.1:26658"
```

Comment:
devnet could be generated from scratch, no need for a fixed configuration

Reply:
I removed it, and use the argument from the command line.